direct.mk

```diff
@@ -141,12 +139,14 @@ help: ## this help
 .PHONY: dep-validate
 dep-validate:
 	@echo "+ $@"
-	$(if $(VNDR), , \
-		$(error Please install vndr: go get github.com/lk4d4/vndr))
-	@rm -Rf .vendor.bak
-	@mv vendor .vendor.bak
-	@$(VNDR)
-	@test -z "$$(diff -r vendor .vendor.bak 2>&1 | tee /dev/stderr)" || \
-		(echo >&2 "+ inconsistent dependencies! what you have in vendor.conf does not match with what you have in vendor" && false)
-	@rm -Rf vendor
-	@mv .vendor.bak vendor
+	@$(MAKE) go-mod-vendor
```

Comment:
Silly question: is this different from adding `go-mod-vendor` after `dep-validate` ? So

```makefile
dep-validate: go-mod-vendor
```

Reply:
Yeah, that makes sense, will change.